cms/static/js/spec/views/course_video_settings_spec.js

```diff
@@ -281,8 +302,286 @@ define(
                 verifyPreferanceErrorState($courseVideoSettingsEl.find('.transcript-languages-wrapper'), false);
             });
 
+            it('shows provider selected view if active provider is present', function() {
+                var $selectedProviderContainerEl = $courseVideoSettingsEl.find('.transcript-provider-wrapper .selected-transcript-provider');
+                expect($selectedProviderContainerEl.find('span').html()).toEqual(courseVideoSettingsView.selectedProvider);
+                expect($selectedProviderContainerEl.find('button.action-change-provider')).toExist();
+                // Verify provider list view is not shown.
+                expect($courseVideoSettingsEl.find('.transcript-provider-wrapper .transcript-provider-group')).not.toExist();
```

Comment:
What exactly we are doing in this test?

Reply:
We are checking that if we have transcript preferences stored already, the view should populate that specific view